docs/source/gcclassic-user-guide/run-checklist.rst

```diff
@@ -55,14 +55,15 @@ Each-time setup
 
    .. attention::
 
-      Prior to running with :option:`GEOS-FP` meteorology, be aware
+      Prior to running with GEOS-FP meteorology, be aware
```

Comment:
Add important note, maybe in its own featured text box:
The convection scheme used for GEOS-FP met generation changed from RAS to Grell-Freidas with impact on GEOS-FP meteorology files starting June 1, 2020, specifically enhanced vertical transport. In addition, there is a bug in convective precipitation flux following the switch where all values are zero. While this bug is automatically fixed by computing fluxes online for runs starting on or after June 1 2020, the fix assumes meteorology year corresponds to simulation year. Due to these issues we recommend splitting up GEOS-FP runs in time such that a single simulation does not run across June 1, 2020. Instead set one run to stop on June 1 2020 and then restart a new run from there. If you wish to use a GEOS-FP meteorology year different from your simulation year please create a GEOS-Chem GitHub issue for assistance.

Reply:
Now fixed in 09a5db3